arm/myoarm_simple.xml

```diff
@@ -0,0 +1,27 @@
+<mujoco model="MyoArm_v0.01">
+<!-- =================================================
+    Copyright 2020 Vikash Kumar, Vittorio Caggiano, Guillaume Durandau
+	Model	:: MyoArm (MuJoCoV3.2.4)
+    Author  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano, Huawei Wang
+    source  :: https://github.com/vikashplus
+    License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

Comment:
some files have License and others do not. Do we need to have this license for each file?